pkg/volume/storageos/storageos_util.go

```diff
@@ -323,27 +346,30 @@ func getLoopDevice(path string, exec mount.Exec) (string, error) {
 	}
 
 	args := []string{"-j", path}
-	out, err := exec.Run(losetupPath, args...)
+	cmd := exec.Command(losetupPath, args...)
+	out, err := cmd.CombinedOutput()
 	if err != nil {
 		glog.V(2).Infof("Failed device discover command for path %s: %v", path, err)
 		return "", err
 	}
 	return parseLosetupOutputForDevice(out)
 }
 
-func makeLoopDevice(path string, exec mount.Exec) (string, error) {
-	args := []string{"-f", "--show", path}
-	out, err := exec.Run(losetupPath, args...)
+func makeLoopDevice(path string) (string, error) {
+	args := []string{"-f", "-P", "--show", path}
+	cmd := exec.Command(losetupPath, args...)
```

Comment:
Please add a comment that you intentionally run where kubelet is (e.g. in kubelet's container).